README.md

```diff
@@ -4,6 +4,13 @@ JSON Schemas for the Planning Inspectorate's Data Model.
 
 All messages over the enterprise service bus are in Data Model compliant formats, the schema's in this repository defined those formats.
 
+## Rules of engagement
+* Messages broadcast over the service bus are ensured to have been pre-validated against the schema. 
+* Each service bus topic corresponds to one schema. 
+* Each message encompasses the complete state of an entity (as defined by [ECST](https://blogs.mulesoft.com/api-integration/strategy/event-carried-state-messages/)) and does not delineate specific changes (the delta). 
+* Consumers can process related messages received through different topics regardless of the order in which they are sent. 
+* Required values cannot be null, while optional values may be null.
```

Comment:
> Required values cannot be null, while optional values may be null. All properties are required to be present in the message, whether null or otherwise.

Reply:
Updated